resources/customized_payloads/F-15ESE.lua

```diff
@@ -2,170 +2,158 @@ local unitPayloads = {
 	["name"] = "F-15ESE",
 	["payloads"] = {
 		[1] = {
-			["displayName"] = "Liberation Strike",
-			["name"] = "Liberation Strike",
+			["displayName"] = "AIM-9M x 4, AIM-120C x 2, CBU-87 x 6, TGP",
```

Comment:
This does nothing in Liberation.

Reply:
Yeah, I dunno why this got added in to the file. This is one of the default F-15E loadouts, I believe. I'll fix it.